ci/bamboo/build-manylinux-prototype.sh

```diff
@@ -1,101 +0,0 @@
-#!/bin/bash
```

Comment:
@scottpurdy, you make a good point:
> The build_manylinux_wheel.sh script isn't really specific to manylinux. It uses that name in a couple places but it is mostly just a bash script for building the wheel and when you run it inside the manylinux Docker container then you get a manylinux wheel.

The script presently has a couple of manylinux docker image specifics that set up the special python from the manylinux image, and everything else is pretty generic.

What do you think about breaking out the generic part into a separate "builder" script and turning  `build_manylinux_wheel.sh` into a thin wrapper that simply sets up the python environment from manylinux docker image and invokes the generic "builder" script?